attachments/LVV-T2116.ipynb

```diff
@@ -0,0 +1,295 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e7b02d5f",
+   "metadata": {},
+   "source": [
+    "## LVV-T2116: Report"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c0f4fe85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.time import Time, TimeDelta\n",
+    "import numpy\n",
+    "import pandas as pd\n",
+    "from matplotlib import pylab as plt\n",
+    "\n",
+    "from lsst_efd_client import EfdClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b8c10ae9",
+   "metadata": {},
+   "source": [
+    "### Process\n",
+    "The process is to interrogate the EFD to see if there are any messages missing over a representative 5 day window of continuous operation.\n",
+    "We will need to connect to the EFD at the summit.  This is done in the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9f9483bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "efd = EfdClient('ldf_stable_efd')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db2bc9d2",
+   "metadata": {},
+   "source": [
+    "The time window for this test was chosen to overlap with the window from LVV-T2115, but shifted by a day to show that in the regions where the windows overlap, the results are the same, but that even outside the window chosen for the summit test we still recover every message.\n",
+    "The start of the window is `2021-05-26T00:00:00`.\n",
+    "\n",
+    "The topic is the same as that used in LVV-T2115 for the same reasons.\n",
+    "I.e. it is the one that exercises the system the most and will, therefore, be the most likely to show failures.\n",
+    "The topic is `lsst.sal.MTM1M3.forceActuatorData`.\n",
+    "\n",
+    "We do the data retrieval in a slightly different way here than on the summit because of a technical issue with using the \"chunked\" method at NCSA.\n",
```

Comment:
Does this have a Jira ticket to track the work necessary to correct this problem or has this already been fixed?

Reply:
I've filed a ticket.  We think it might be a timeout issue, but will have a look [DM-31913](https://jira.lsstcorp.org/browse/DM-31913).  It has not been fixed as far as I know.

Reply:
OK. As long as a ticket is filed and hopefully gets looked at in the future, I'm fine here.